spec/adaptors/quickbooks_online/operations/bill_payment/find_spec.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+support :operation_shared_examples,
+        :quickbooks_online_helpers
+
+RSpec.describe LedgerSync::Adaptors::QuickBooksOnline::BillPayment::Operations::Find do
+  include QuickBooksOnlineHelpers
+
+  let(:vendor) do
+    LedgerSync::Vendor.new(ledger_id: '123')
```

Comment:
I get that if we wanna mock better QBO response that sends also `name` attribute with `value` in `VendorRef`. But that may not be same depending on a factory. In case we do some internal caching of existing factories and we will sequence name automaticaly, etc.

Reply:
Allrite; all good here, scratch that last comment 👌